### PR TITLE
📖Add info about usage of kubectl vc exec

### LINKED ISSUE #169: how to re-create vc1-kubeconfig file ?
**User Story**

I was guide 
`kubectl vc create -f https://raw.githubusercontent.com/kubernetes-sigs/cluster-api-provider-nested/master/virtualcluster/config/sampleswithspec/virtualcluster_1_nodeport.yaml -o vc-1.kubeconfig`

to create vc-1.kubeconfig file , for whatever reason , the file is deleted, how can I recreate it from existing cluster? seems at least a doc is helpful here if the feature is already there

As a [developer/user/operator] I would like to [high level description] for [reasons]

**Detailed Description**

[A clear and concise description of what you want to happen.]

**Anything else you would like to add:**

[Miscellaneous information that will assist in solving the issue.]

/kind feature

### PR DESCRIPTION
**Pull request by jichenjc**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #169

### CODE REVIEW

File: virtualcluster/doc/demo.md [hunk 1]
```diff
@@ -323,6 +323,37 @@ Load average: 0.45 0.47 0.54 1/1308 23
   PID  PPID USER     STAT   VSZ %VSZ CPU %CPU COMMAND
 ```
 
+## (Optional) use `kubectl vc exec` to enter cluster context and regenerate kubeconfig for paticular cluster
+
+You can use `kubectl vc exec` to operator on desired cluster, for example:
```
Comment (christopherhein) [inline virtualcluster/doc/demo.md:328]:
```suggestion
You can use `kubectl vc exec` to operate on desired virtualcluster, for example:
```

File: virtualcluster/doc/demo.md [hunk 2]
```diff
@@ -323,6 +323,37 @@ Load average: 0.45 0.47 0.54 1/1308 23
...
+```
+
+Then you can see the pods in that cluster (this example we have a `test-deploy-5f4bcd8c-4nfxg` pod):
+```bash
+# kubectl get pods
+NAME                         READY   STATUS    RESTARTS   AGE
+test-deploy-5f4bcd8c-4nfxg   1/1     Running   0          42h
+```
+
+Also, the kubeconfig is generated at `/home/cloudusr/.kube/vc/` with name `<namespace>-<cluster name>.kubeconfig`, in our example, it'
```
Comment (christopherhein) [inline virtualcluster/doc/demo.md:347]:
```suggestion
Also, the kubeconfig is generated and stored at `~/.kube/vc/` with name `<namespace>-<cluster name>.kubeconfig`, in our example, it's
```

File: virtualcluster/doc/demo.md [hunk 3]
```diff
@@ -323,6 +323,37 @@ Load average: 0.45 0.47 0.54 1/1308 23
...
+ `/home/cloudusr/.kube/vc/default-b9bf25-vc-sample-1.kubeconfig`, you can use it if the original kubeconfig is lost.
```
Comment (christopherhein) [inline virtualcluster/doc/demo.md:348]:
```suggestion
 `~/.kube/vc/default-b9bf25-vc-sample-1.kubeconfig`, you can use it if the original kubeconfig is lost.
```

**Review by christopherhein — CHANGES_REQUESTED** [review]
couple of small grammatical issues otherwise looks great.

File: virtualcluster/doc/demo.md [hunk 4]
```diff
@@ -323,6 +323,37 @@ Load average: 0.45 0.47 0.54 1/1308 23
   PID  PPID USER     STAT   VSZ %VSZ CPU %CPU COMMAND
 ```
 
+## (Optional) use `kubectl vc exec` to enter cluster context and regenerate kubeconfig for paticular virtualcluster
```
Comment (christopherhein) [inline virtualcluster/doc/demo.md:326]: missed one more thing, thank you!

```suggestion
## (Optional) use `kubectl vc exec` to enter cluster context and regenerate kubeconfig for particular virtualcluster
```

Comment (jichenjc) [reply virtualcluster/doc/demo.md]: done :)

**Pull request merged**